libraries/chain/controller.cpp

```diff
@@ -3454,7 +3454,7 @@ void controller::add_to_ram_correction( account_name account, uint64_t ram_bytes
       } );
    }
 
-   if (auto dm_logger = get_deep_mind_logger()) {
+   if (auto dm_logger = get_deep_mind_logger(false)) {
```

Comment:
We do not want to log this for dry-run transactions.

Reply:
This is only called by `apply_context::schedule_deferred_transaction`

Reply:
Please add a comment indicating that.